src/components/ThemeToggle.tsx

```diff
@@ -12,13 +13,21 @@ const ThemeToggle = () => {
   };
 
   return (
-    <div onClick={handleChangeTheme}>
-      <img
-        src={theme === "light" ? halfMoon : sheep}
-        alt={"Moon Icon for Theme Toggle"}
-        width={"50"}
-      />
-    </div>
+    <button className={styles.button} onClick={handleChangeTheme}>
+      {theme == "light" ? (
+        <img
+          src={halfMoon}
```

Comment:
Why you would prefer two image tags over one?

Reply:
It seemed more readable to me. The alternative is

```js
      <img
        src={theme == "light" ? halfMoon : sun}
        alt={
          theme == "light"
            ? "Moon Icon for Theme Toggle"
            : "Sun Icon for Theme Toggle"
        }
        title={
          theme == "light"
            ? "Zum dunklen Design wechseln"
            : "Zum hellen Design wechseln"
        }
      />
```

Reply:
I agree with Aloso - two tags are more readable.

Reply:
You could also have two functions return the corresponding values similar to [leos new PR](https://github.com/LeoTuet/ui-we-want-to-sleep/pull/70/commits/0d57d89792eb3fd919961c830bb00cb8f00c46e9#diff-49b1b26f2b78139b3a19a58f388ed5ad7b91e9d4eeb736e19402c8daf55051d5R31)

Reply:
I changed it now.